pages/services/beta-storage/0.5.3-beta/install/artifacts-container-based/index.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: layout.pug
+navigationTitle: Via Artifacts Container
+title: Via Artifacts Container
+menuWeight: 30
+excerpt: Installing the DC/OS Storage Service using the artifacts container
+enterprise: true
+beta: true
+---
+#include /services/include/beta-software-warning.tmpl
+
+<p class="message--note"><strong>NOTE: </strong> This install method is <strong>deprecated</strong>.
+Please consider switching to the <a href="../package-registry-based/">package registry based install</a> method.</p>
+
+# Prerequisites
+
+- DC/OS Enterprise 1.12 or later.
+- [DC/OS CLI](/latest/cli/install/) is installed and you are logged in as a superuser.
+
+# Download the package tarball
+
+DC/OS Storage Service (DSS) is packaged as a tarball.
+The tarball can be downloaded from [Mesosphere support site](https://support.mesosphere.com/hc/en-us/articles/213198586). Please note that in order to download software from the support page, you must have a Mesosphere support account.
+
+# Build the local universe and artifacts server
+
+Extract DC/OS Storage Service (DSS) package tarball.
+You should name the tarball `package.tgz`.
+
+<p class="message--note"><strong>NOTE: </strong> Do not delete the package after extraction.</p>
+
+```bash
+$ tar zxvf package.tgz
+$ cd package
+```
+
+Build the Docker image that will serve both the local universe and the artifacts used by DC/OS Storage Service.
+Specify the `DOCKER_IMAGE` environment variable accordingly so that the image can be pushed to docker hub or your local docker registry.
+
+<p class="message--note"><strong>NOTE: </strong>Do not specify the tag of the Docker image.
+The tag will be generated by the build script according to the version of the DSS package. </p>
+
+```bash
+$ DOCKER_IMAGE=user/storage-artifacts universe/build.sh
+```
+
+The build script will generate some instructions for the next steps like the following.
+Notice, that the `<SHA>` will be replaced with a real SHA1 value, reflecting the version of the DSS package.
+
+<p class="message--note"><strong>NOTE: </strong>* Do not change the SHA value during the instruction workflow.</p>
+
+```bash
```

Comment:
Why are these instructions formatted as code?

Reply:
They're formatted this way because it's a snapshot of script output. Is there a different notation, other than `bash` that we should use to reflect this situation?